lua/efmls-configs/formatters/sql-formatter.lua

```diff
@@ -0,0 +1,23 @@
+-- Metadata
+-- languages: sql
+-- url: https://github.com/sql-formatter-org/sql-formatter
+
+local fs = require('efmls-configs.fs')
+local ok, Path = pcall(require, 'plenary.path')
```

Comment:
Should I mark plenary as an dependency on the readme? I just used it because I saw it in the code for rustfmt.